### PR TITLE
fix for laser data

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

**Review by mslupeck — APPROVED** [review]

**Comment by AllaMaevskaya** [comment]
build/O2/fullCI  error not relate to my code

**Review by davidrohr — COMMENTED** [review]
Should we generally disable this?
If this is corrupt data, do you want to at least print an alarm in case of non-laser data?

File: Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx [hunk 1]
```diff
@@ -63,8 +63,6 @@ void FT0ChannelTimeTimeSlotContainer::fill(const gsl::span<const FT0CalibrationI
       mHistogram[chID]->Fill(chTime);
       ++mEntriesPerChannel[chID];
       LOG(debug) << "@@@@entries " << mEntriesPerChannel[chID] << " chID " << int(chID) << " time " << chTime << " tiestamp " << uint64_t(entry.getTimeStamp());
-    } else {
-      LOG(fatal) << "Invalid channel data";
```
Comment (shahor02) [inline Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx:67]: is this protection against coding error or corrupted data? Normally, the latter should not lead to fatal but raise an alarm (LOG(alarm)) and pass over it.

Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx:67]: This is protection was done before look-up table implementation, now its not actual. Later reading of dead channel map will be add

Comment (shahor02) [reply Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx:67]: So, shall I close this PR?

Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx:67]: Please  merge it. We have problem to test calibration @P2 with laser data with addition channels for reference PMTs.

**Pull request merged**